azkaban/__main__.py

```diff
@@ -324,6 +324,8 @@ def _callback(cur_bytes, tot_bytes, file_index, _stdout=sys.stdout):
         session.create_project(name, name)
       else:
         raise err
+    except HTTPError as e:
```

Comment:
`HTTPError` is a very generic error; is it possible to tighten the check? For example checking the response's status (is it 404?).

Reply:
Completely valid point. I have updated the code so that it only reacts with a create project when a `410` is returned. This is the http code returned by Azkaban 3.59.0 and above when attempting to upload to a project that does not exist. I have validated the code works locally before I pushed to my branch..